backend/src/main/java/ai/verta/modeldb/experiment/ExperimentServiceImpl.java

```diff
@@ -1423,23 +1355,4 @@ public void deleteExperiments(
           responseObserver, e, DeleteExperiments.Response.getDefaultInstance());
     }
   }
-
-  private boolean deleteExperiments(List<String> experimentIds)
-      throws InvalidProtocolBufferException {
-
-    List<String> accessibleExperimentIds =
-        getAccessibleExperimentIDs(experimentIds, ModelDBServiceActions.UPDATE);
-
-    if (accessibleExperimentIds.isEmpty()) {
-      String errorMessage =
-          "Access is denied. User is unauthorized for given Experiment IDs : "
-              + accessibleExperimentIds;
-      ModelDBUtils.logAndThrowError(
-          errorMessage,
-          Code.PERMISSION_DENIED_VALUE,
-          Any.pack(DeleteExperiments.getDefaultInstance()));
-    }
-
-    return experimentDAO.deleteExperiments(accessibleExperimentIds);
-  }
```

Comment:
I moved this validation from here to DAO due to this https://github.com/VertaAI/modeldb/pull/554/files#r412863905 code.